TombEngine/Renderer/Native/DirectX11/DX11GraphicsDevice.cpp

```diff
@@ -0,0 +1,1017 @@
+#include "framework.h"
+
+#ifdef SDL_PLATFORM_WIN32
```

Comment:
Why this particular file is completely under this `ifdef`, but other DirectX-specific ones are not?